pyart/graph/radardisplay.py

```diff
@@ -586,14 +617,47 @@ def plot_range_ring(self, range_ring_location_km, npts=100, ax=None):
             Number of points in the ring, higher for better resolution.
         ax : Axis
             Axis to plot on.  None will use the current axis.
+        col : str or value
+            Color to use for range rings.
+        ls : str
+            Linestyle to use for range rings.
 
         """
-        ax = self._parse_ax(ax)
+        ax = parse_ax(ax)
         theta = np.linspace(0, 2 * np.pi, npts)
         r = np.ones([npts], dtype=np.float32) * range_ring_location_km
         x = r * np.sin(theta)
         y = r * np.cos(theta)
-        ax.plot(x, y, 'k-')
+        if lw is None:
+            lw = 2
+        if ls is None:
+            ls = '-'
+        if col is None:
+            col = 'k'
```

Comment:
Default values should be specified explicitly in the function signature not by checking for None.